src/orientation-arm-model.js

```diff
@@ -13,7 +13,9 @@
  * limitations under the License.
  */
 
-const HEAD_ELBOW_OFFSET = new THREE.Vector3(0.155, -0.465, -0.15);
+const HEAD_ELBOW_OFFSET_RIGHTHANDED = new THREE.Vector3(0.155, -0.465, -0.15);
+const HEAD_ELBOW_OFFSET_LEFTHANDED = new THREE.Vector3(-0.155, -0.465, -0.15);
+let head_elbow_offset = HEAD_ELBOW_OFFSET_RIGHTHANDED;
```

Comment:
i'd change this to be `camelBack`-cased (i.e., `headElbowOffset`) like the other local variables

Reply:
Agree